source/extensions/omni.isaac.lab/omni/isaac/lab/sim/simulation_context.py

```diff
@@ -225,6 +225,16 @@ def __init__(self, cfg: SimulationCfg | None = None):
         else:
             self._app_control_on_stop_handle = None
 
+        # set denoiser mode
+        try:
+            import omni.replicator.core as rep
+
+            rep.settings.set_render_rtx_realtime(antialiasing=self.cfg.antialiasing_mode)
```

Comment:
This setting gets overwritten to `DLAA` inside camera and tiledcamera is the width or height is small. Not sure if this is desirable there.

Reply:
that's intentional because we need the DLAA hack to avoid frame delay on small images.